chain-signatures/node/src/metrics.rs

```diff
@@ -398,6 +398,15 @@ pub(crate) static SIGNATURE_PUBLISH_RESPONSE_ERRORS: Lazy<CounterVec> = Lazy::ne
     .unwrap()
 });
 
+pub(crate) static PROTOCOL_LATENCY_ITER_CNT: Lazy<CounterVec> = Lazy::new(|| {
+    try_create_counter_vec(
```

Comment:
Do you want to measure latency or count iterations? Let's rename this accordingly and use counter if it's the later one.